frontend/cypress/integration/mainnet/mainnet.spec.ts

```diff
@@ -12,30 +12,50 @@ describe('Mainnet', () => {
         Cypress.Commands.add('waitForBlockData', () => {
             cy.wait('@tx-outspends');
             cy.wait('@pools');
-          });
+        });
     });
 
     it('loads the status screen', () => {
-      cy.visit('/status');
-      cy.get('#mempool-block-0').should('be.visible');
-      cy.get('[id^="bitcoin-block-"]').should('have.length', 8);
-      cy.get('.footer').should('be.visible');
-      cy.get('.row > :nth-child(1)').invoke('text').then((text) => {
-        expect(text).to.match(/Tx vBytes per second:.* vB\/s/);
-      });
-      cy.get('.row > :nth-child(2)').invoke('text').then((text) => {
-          expect(text).to.match(/Unconfirmed:(.*)/);
-      });
-      cy.get('.row > :nth-child(3)').invoke('text').then((text) => {
-        expect(text).to.match(/Mempool size:(.*) (kB|MB) \((\d+) (block|blocks)\)/);
-      });
+        cy.visit('/status');
+        cy.get('#mempool-block-0').should('be.visible');
+        cy.get('[id^="bitcoin-block-"]').should('have.length', 8);
+        cy.get('.footer').should('be.visible');
+        cy.get('.row > :nth-child(1)').invoke('text').then((text) => {
+            expect(text).to.match(/Tx vBytes per second:.* vB\/s/);
+        });
+        cy.get('.row > :nth-child(2)').invoke('text').then((text) => {
+            expect(text).to.match(/Unconfirmed:(.*)/);
+        });
+        cy.get('.row > :nth-child(3)').invoke('text').then((text) => {
+            expect(text).to.match(/Mempool size:(.*) (kB|MB) \((\d+) (block|blocks)\)/);
+        });
     });
 
+
     it('loads the dashboard', () => {
-      cy.visit('/');
-      cy.waitForSkeletonGone();
+        cy.visit('/');
+        cy.waitForSkeletonGone();
     });
 
+    it('loads skeleton when changes between networks', () => {
+        cy.visit('/');
+        cy.waitForSkeletonGone();
+
+        changeNetwork("testnet");
+        changeNetwork("signet");
+        changeNetwork("liquid");
+    });
+
+    function changeNetwork (network: "testnet"|"signet"|"liquid" ){
```

Comment:
@MiguelMedeiros thanks for adding a test! Custom commands that are shared across specs should go into `support/commands.ts` like this:

https://github.com/mempool/mempool/blob/master/frontend/cypress/support/commands.ts#L49-#L53

see https://docs.cypress.io/api/cypress-api/custom-commands for reference

Reply:
Fixed here:
https://github.com/mempool/mempool/pull/696/commits/b4525b05f6d4fb422d4aeed0f446081a23e56986